docs/extras/data-sources/index.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to the data
+
+OpenTrials is a database of information on clinical trials. The database *threads together* information on trials, collected from a wide range of sources, in a single centralised database, and thereby provides value-add for analysis of the data on clinical trials that has not been possible, or has been inaccessible, with publication practices for clinical trials to date.
+
+Here, we'll give a brief rundown of the data available in OpenTrials, and why it is important.
+
+## Clinical trial registries
+
+Data from clinical trial registries forms the core of the OpenTrials database. These registries have metadata about trials, providing, for example, information on the condition(s) being targeted, the intervention(s) being tested, the number and gender of participants, and so on. We pull in this metadata, look for matches across the registers (multiple "records" talking about the same trial), and build an initial thread of public information about trials from these registriess.
+
+One of the primary registers is [ClinicalTrials.gov](https://clinicaltrials.gov). Have a look around there for an understanding of type of information that we get from registries.
+
+## Pubmed
+
+[Pubmed](http://www.ncbi.nlm.nih.gov/pubmed) provides an API exposing over 26 million citations in the biomedical literature. In OpenTrials, we use PubMed as a way to find academic publications that reference clinical trials. This allows us to provide direct links from a trial page, to academic papers that address that trial in some way (usually either introducing a trial and explaining how it will be carried out, or reporting the results of a completed trial).
+
+## FDA
+
+The [FDA](http://www.fda.gov) provides data related to foods and drugs available on the US market. It is a veritable treasure trove of information, and important aspects of this body of knowledge are usable in OpenTrials.
+
+We take data from the FDA in two different "collectors".
+
+First, we use an FDA API to extract information on drug names. This data is used to help normalise our "interventions" data.
+
+Second, we [scrape Drug Approval Packages](http://opentrials.net/2016/08/10/opentrialsfda-unlocking-the-trove-of-clinical-trial-data-in-drugsfda/) (DAP) data from the FDA. Unfortunately, there is no API for this data, and DAP data tends to be in unsearchable, scanned PDF files. However, they are a very high quality source of information on clinical trials, containing detailed information on the methods and results of trials, and required as part of a drug's marketing authorisation (i.e. for a company to get permission to sell the drug, assuming it meets certain standards). Typically the information contained in DAPs is less biased than results of clinical trials which are published in academic journals, so are of interest to OpenTrials and its users.
```

Comment:
Might be useful to link to Erick's article where he analyzed the results from publications and FDA data, showing the differences between them.